pandas/core/groupby/groupbyindexing.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import annotations
```

Comment:
this file should just be called `indexing.py`

Reply:
Personally I try to avoid identical file names in different folders. It is difficult to distinguish in editor tabs.
However, I am happy to change the name.